test/integration/policy_gatekeeper_operator_test.go

```diff
@@ -241,7 +241,7 @@ var _ = Describe("", Ordered, Label("policy-collection", "community"), func() {
 			By("Creating invalid namespace on managed")
 			Eventually(func() interface{} {
 				out, _ := utils.KubectlWithOutput("apply", "-f", "../resources/gatekeeper/ns-create-invalid.yaml", "--kubeconfig="+kubeconfigManaged)
-				if out == "namespace/e2etestfail created" {
+				if strings.HasPrefix(out, "namespace/e2etestfail created") {
```

Comment:
Would `strings.Contains` be a little more resilient?

Reply:
True--I'll update it. I wasn't 100% sure this string wasn't contained somehow within the violation message, but I confirmed it's not.